.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: Continuous Integration Pipeline
+
+on:
+  push:
+    branches:
+      - '**'
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'LICENSE'
+  pull_request:
+    branches:
+      - '**'
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '.gitignore'
+      - 'LICENSE'
+
+env:
+  NODE_VERSION: '16'
+
+jobs:
+  continuous-integration:
```

Comment:
I think we should align our workflows - please change this to "test" to match the job on npm "test:coverage", and then change the name of the file to continuous-integration.yml

Reply:
Thank you @timler I have updated my PR